packages/data-service/src/data-service.ts

```diff
@@ -1493,27 +1529,41 @@ class DataService extends EventEmitter {
   /**
    * Create a ClientSession that can be passed to commands.
    */
-  startSession(): ClientSession {
-    return this._initializedClient.startSession();
+  startSession(clientType: ClientType): ClientSession {
```

Comment:
Would it help the typing here if we have a `ClientTypeSession` (other names?) type that extends the `ClientSession` with the `clientType` as a key? Then in `killSessions` we might not need to do `any` casting.

Reply:
Sure, done! :)